code/modules/projectiles/gun.dm

```diff
@@ -854,8 +854,26 @@
 		actual_angle -= 360
 	if(total_recoil > 0)
 		recoil_camera(user, total_recoil + 1, (total_recoil * recoil_backtime_multiplier)+1, total_recoil, actual_angle)
+		var/consequences_of_our_actions = (HAS_TRAIT(user, TRAIT_EASYLIMBDISABLE)? 2 : 4)
```

Comment:
This mechanic is... questionable. I don't see it really being beneficial - if it happens at all, it should only happen on un-wielded very large calibre firearms. But definitely not added in a ruin PR.